app/controllers/api/v1/admins_controller.rb

```diff
@@ -0,0 +1,48 @@
+class AdminsController < ApplicationController
+  def new
+    @admin = Admin.new
+  end
+
+  def index
+    @admin = Admin.all
+  end
+
+  def show
+    @admin = Admin.find params[:id]
+  end
+
+  def create
+    @admin = Admin.new params(admin_params)
+    if @admin.save
+      redirect_to admins_path
```

Comment:
let's render JSON! We are creating a JSON Api here to be used by our react app:

```render json: @admin```

so change all the renders and redirects to return JSON